.github/workflows/kotlin.yml

```diff
@@ -79,15 +96,15 @@ jobs:
       - name: Cache build artifacts
         uses: actions/cache@v1
         with:
-            path: ~/.gradle/caches
-            # Don't set restore-keys so cache is always only valid for the current build config.
-            # Also don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
-            key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
+          path: ${{ env.GRADLE_CACHE_PATH }}
+          # Don't set restore-keys so cache is always only valid for the current build config.
+          # Also don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
+          key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
```

Comment:
I guess we don't need the `restore-key` here since the prior steps would have populated it?

Reply:
Yes. This is documented on line 47.